src/pages/blog/basic_linear_modeling.md

```diff
@@ -0,0 +1,1185 @@
+---
+layout: "../../layouts/BlogPage.astro"
+title: "Reaching Insights about Stuff: Getting A Broad Picture With Linear Modeling"
+authors: 
+- name: "glumbaron"
+date: "January 22, 2023"
+description: "Let's use basic linear modeling to get a sense of how player attributes affect performance!"
+tags: []
```

Comment:
Suggestion: Add tags to this post
If you are happy without any tags, lmk and ill merge